src/ducks/classrooms.js

```diff
@@ -1,11 +1,13 @@
 import { State, Effect, Actions } from 'jumpstate';
 import PropTypes from 'prop-types';
-import { get } from '../lib/edu-api';
+import { get, post, httpDelete } from '../lib/edu-api';
```

Comment:
[minor, irrelevant] I understand why this has to break the naming convention and be called `httpDelete` instead of `delete`, but it still makes me sad. 😑